client/state/nps-survey/actions.js

```diff
@@ -139,3 +161,24 @@ export function submitNpsSurveyWithNoScoreRequestFailure( err ) {
 		error: err,
 	};
 }
+
+export function sendNpsSurveyFeedbackRequesting( surveyName, feedback ) {
+	return {
+		type: NPS_SURVEY_SEND_FEEDBACK_REQUESTING,
+		surveyName,
+		feedback,
+	};
+}
+
+export function sendNpsSurveyFeedbackSuccess() {
+	return {
+		type: NPS_SURVEY_SEND_FEEDBACK_REQUEST_SUCCESS,
+	};
+}
+
+export function sendNpsSurveyFeedbackFailure( err ) {
+	return {
+		type: NPS_SURVEY_SEND_FEEDBACK_REQUEST_FAILURE,
+		error: err,
```

Comment:
We can consider to rename `err` to `error` so this can be simplify as
```
return {
    type: NPS_SURVEY_SEND_FEEDBACK_REQUEST_FAILURE,
    error,
};
```